roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- import_tasks: cluster_health.yml
+  vars:
+    l_check_bricks: true
+  when: glusterfs_is_native
+
+- import_tasks: heketi_db_backup.yml
```

Comment:
`ERROR! Unable to retrieve file contents
Could not find or access '/home/travis/build/openshift/openshift-ansible/playbooks/openshift-glusterfs/private/heketi_db_backup.yml' on the Ansible Controller.`

Reply:
Done